README.md

```diff
@@ -655,6 +656,7 @@ Returns a paginated result set.
 | Name | Type | Description |
 | ---- | ---- | ----------- |
 | tracked | boolean | Whether the user is tracked for the course |
+| roles | [ string ] | Accepted values: "learner", "author", "contributor", "publisher" |
```

Comment:
So we have course roles and org roles. The table above explains org roles well. Course roles above, however, are not explained well. Do we want a table for them? also do we want to explicitly label them as course vs org roles? I can give you text for the above four course roles.

Reply:
👍 good idea

Reply:
"learner" - A learner a learner in the course
 "author" - An author an org member with author privileges
 "contributor" - A contributor  an out-of-org user with author privileges
"publisher" - A publisher can publish newer revisions of the course - all learners only see the latest revision.

Reply:
"author" and "contributor" are synonyms, org membership doesn't enter into it...?

Reply:
For clarity, should I just merge this PR and add the roles table in a separate PR once the final table verbiage is set?

Reply:
oh its fine.. lets go with

"author" / "contributor" can edit the course
"learner" can learn the course
"publisher" .. above

Please make updates and 🚢

Reply:
See latest commit for "Roles" updates. Let me know if anything needs to be adjusted further.